Week5/app/src/main/java/com/isabelledionisius/mymovieapp/MovieListViewHolder.kt

```diff
@@ -0,0 +1,18 @@
+package com.isabelledionisius.mymovieapp
+
+import android.media.Image
+import android.view.View
+import android.widget.ImageView
+import android.widget.TextView
+import androidx.recyclerview.widget.RecyclerView
+
+// passing itemview to the superclass constructor
+class MovieListViewHolder(itemView: View): RecyclerView.ViewHolder(itemView) {
+    /**
+     * View holder items for MovieAdapter class
+     */
+
+    var titleTextView = itemView.findViewById<TextView>(R.id.title)
```

Comment:
again, avoid `findViewById()`. You can show data here, instead of in the adapter